plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/weighers/PriorityWeigher.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2000-2021 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
+package org.jetbrains.kotlin.idea.completion.weighers
+
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.codeInsight.lookup.LookupElementWeigher
+import org.jetbrains.kotlin.idea.completion.ItemPriority
+import org.jetbrains.kotlin.idea.completion.priority
+
+internal object PriorityWeigher {
+    const val WEIGHER_ID = "kotlin.priority"
```

Comment:
Do we need to have this property public?

Reply:
I don't need it now, But it could potentially be used if we want to rearrange the weigers in https://github.com/JetBrains/intellij-community/blob/838bf7cbdcf0e279efc218b832345efe48e16b56/plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/weighers/Weighers.kt#L11. If you like I can change all the current weigers to have this piece private unless it's used in `Weighers.kt`.